app/create_db.py

```diff
@@ -2,6 +2,7 @@
 
 import re
```

Comment:
I think this import can be removed now that we are using regex.

Reply:
good catch, missed that one